.changes/allow-recursive-asset-scope-on-file-drop-directory.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri": 'minor:breaking'
```

Comment:
i'd say this qualifies as patch:bug, no? (i'm super bad at this, so please wait for feedback from others 😂 )

Reply:
probably just an enhancement
```suggestion
"tauri": 'patch:enhance'
```